### PR TITLE
Remove dead CSS link

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: benchto-service/src/main/webapp/index.html [hunk 1]
```diff
@@ -34,7 +34,6 @@
     <script src="js/filters.js"></script>
     <script src="js/main.js"></script>
 
-    <link rel="stylesheet" href="http://google-code-prettify.googlecode.com/svn/trunk/src/prettify.css">
```
Comment (fiedukow) [inline benchto-service/src/main/webapp/index.html]: Shouldn't that be replaced by this?
https://cdn.rawgit.com/google/code-prettify/master/src/prettify.css

Comment (findepi) [inline benchto-service/src/main/webapp/index.html]: First, I don't think the app needs this in any way.
Second, http://google-code-prettify.googlecode.com/ directs me to https://github.com/google/code-prettify which instructs me to use `<script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js"></script>` ... which the app already does (same file, 23 lines earlier)

**Comment by ArturGajowy** [comment]
`frontend_developers_count += 2 // yay! \o/`

**Comment by fiedukow** [comment]
So now `frontend_developer_count == "12"`?

**Pull request merged**